### DISCUSSION TITLE
How to open about:config with the SPC help?

### DISCUSSION THREAD

**Discussion by iG8R** [YoshifumiFuyuno/Search-from-Popup-or-ContextMenu#82] [Q&A] [answered]
Hi.
Could you, please, help with the method how to open `about:config` page through the SPC?

**Comment by YoshifumiFuyuno** [comment, marked as answer]
Hi.
Sorry for the late reply.
In Firefox, some URLs cannot be opened from extensions.
This is a Firefox specification.

Best regards

https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/tabs/create
> For security reasons, in Firefox, this may not be a privileged URL. So passing any of the following URLs will fail:
> *    chrome: URLs
> *    javascript: URLs
> *    data: URLs
> *    file: URLs (i.e., files on the filesystem. However, to use a file packaged inside the extension, see below)
> *    privileged about: URLs (for example, about:config, about:addons, about:debugging). Non-privileged URLs (e.g., about:blank) are allowed.
> *    The New Tab page (about:newtab) can be opened if no value for URL is provided.

**Reply by iG8R** [reply]
Thanks for the clarification.
